Exercise/02/arrays.py

```diff
@@ -0,0 +1,40 @@
+class ArrayTasks:
+	def __init__(self, arr) -> None:
+		self.arr = arr
+
+	def element_at(self, index) -> int:
+		if index < 0:
+			return "Index out of range"
+		try:
+			return self.arr[index]
+		except IndexError:
+			return "Index out of range"
+	
+	def inclusive_range(self, start_pos, end_pos) -> list:
+		if (start_pos < 0) or (end_pos > len(self.arr)) or (start_pos > end_pos):
+			return "Index out of range"
+		try:
+			return self.arr[start_pos:end_pos+1]
```

Comment:
doesn't make sense to have a if condition and the try-catch block that throws the same error